src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -6124,6 +6124,7 @@ regNumber CodeGen::genGetZeroReg(regNumber initReg, bool* pInitRegModified)
 //                                                       start zero initializing memory.
 //    initReg          - A scratch register (that gets set to zero on some platforms).
 //    pInitRegModified - Sets a flag that tells the callee whether or not the initReg register got zeroed.
+//                       'true' if this method sets initReg to non-zero value. Else 'false'.
```

Comment:
It's slightly different than this. `*pInitRegModified` is set to `true` if `initReg` is set to a non-zero value, `false` if it is set to zero, and not changed if `initReg` is unused.